api/src/pcapi/core/bookings/validation.py

```diff
@@ -130,7 +130,7 @@ def check_is_usable(booking: Booking) -> None:
         forbidden.add_error("payment", "Cette réservation a été remboursée")
         raise forbidden
 
-    if booking.isUsed or booking.status is BookingStatus.USED:
+    if booking.status is BookingStatus.USED:
```

Comment:
Pourquoi on check pas booking.is_used ? le cas REIMBURSED ne peut pas arriver ?

Reply:
Le cas du REIMBURSED est checké avant on dirait.
La question que je me pose, c'est est-ce qu'on est sûr de la cohérence des données entre présence d'un paiement en base et statut `REIMBURSED`?
En effet la fonction `finance_repository.has_reimbursement` stipule `"""Return whether the requested booking has been reimbursed."""` donc ça doit être bon 🤷‍♂️

Reply:
Confiance dans `REIMBURSED` < confiance dans `finance_repository.has_reimbursement` :)